fixham/internal/client/client.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/fixham/internal/docker"
+)
+
+type Client struct {
```

Comment:
What is this a client for? A client generally implies there is a server or service of some sort that this is a frontend for accessing, but I don't think that's the case here.

Reply:
Removing it. Also based on the suggested structure, this will be unnecessary